src/server.tsx

```diff
@@ -77,6 +78,11 @@ const componentAsResponse = (componentWrapper: SlotComponent, meta: EpicTestTrac
     };
 };
 
+const buildHardcodedTests = async (): Promise<Test[]> => {
```

Comment:
Really minor but I wonder whether `buildHardcodedTests` could live under `./tests` instead? Perhaps defined in `./tests/index.ts`? Just to keep the clutter out of server.tsx.

Reply:
What about putting it under the API file? I know it is not actually from the API, but conceptually if we think of the API are returning test data, it works.

Reply:
I've renamed this function to `getAllHarcodedTests` to clarify, and moved it to an index file under `/tests` - hopefully this will make it clear that tests under this folder should be accessed via the interface provided by this function.